scripts/3rdparty.py

```diff
@@ -320,7 +320,7 @@ def makeenv(envlist):
             '--with-pic',
             '--enable-static',
         ])), logfile)
-    execute('make -j', logfile)
+    execute('make', logfile)
```

Comment:
`make -j` is significantly faster for some of the dependencies, at least on on my machine. I'd prefer to keep it at least on Linux.

Can we do one of the following?

1. Detect whether `make` is GNU make and use `-j` if it is?

2. Or detect whether `gmake` is available on the system and use `gmake -j` if it is and `make` if it's not?

3. Or just always require `gmake` to be available (it would be a simpler approach but I'm not sure if gmake is available everywhere by default, e.g. on Alpine and FreeBSD. If it's not, this approach is not preferred).

Reply:
4. Determine sensible amount of threads to use (# of CPU cores) and pass it to -j. BSD make supports this option, just not without an argument. 

Just requiring `gmake` would also be fine by me, but unfortunately many Linux distros don't ship it (as in a symlink to `make`)

Reply:
> Determine sensible amount of threads to use (# of CPU cores) and pass it to -j. BSD make supports this option, just not without an argument.

Great, this is the best option I think.

Reply:
#310

Reply:
Merged into develop.